piccolo_api/crud/endpoints.py

```diff
@@ -529,6 +543,27 @@ async def get_all(
         else:
             query = self.table.select(exclude_secrets=self.exclude_secrets)
 
+        visible_fields = split_params.visible_fields
+        if visible_fields:
+            all_columns_map: t.Dict[str, Column] = {
+                key: value
+                for key, value in zip(
+                    [i._meta.name for i in self.table._meta.columns],
+                    self.table._meta.columns,
+                )
+            }
```

Comment:
Can you do this instead?

```python
all_columns_map = {
    i._meta.name: i for i in self.table_meta.columns
}
```

Reply:
You're right. It’s a lot simpler and better.